components/generic/ImageWithAttribution.vue

```diff
@@ -1,60 +1,106 @@
 <template>
-  <div>
-    <figure>
```

Comment:
The full image with attribution comprises a figure with an image and figcaption. Without figure, figcaption has no bounding context.